components/Banner/src/MDCBannerView.h

```diff
@@ -16,6 +16,14 @@
 
 #import "MaterialButtons.h"
 
```

Comment:
Had you considered exposing a “layout” object property and providing several pre-built layout objects that can be assigned to it? This would allow clients the flexibility to create entirely custom layouts while still benefiting from the structure of our component + out of the box layouts.

Reply:
Previously, there was a layout class built for this purpose, but removed because of the complexity/extra layer of abstraction. I feel introducing it later when we have more user data and use cases would be an ideal case to make it more flexible and extendable. Since the current API meets the needs and can be easily extended later on because the core concept is the same. How do you feel?

Reply:
Makes sense to me! Thanks for the context :)